new-from-ui.go

```diff
@@ -0,0 +1 @@
+package main
```

Comment:
language: Go
<!-- lookout footnote separator -->
_Comment made by the analyzer Dummy._